### DISCUSSION TITLE
how to create new plots

### DISCUSSION THREAD

**Discussion by leochen12-rgb** [ericaltendorf/plotman#881] [General]
update  chia 1.20 add NFT , how to plotman plot new plots . 
Please give the usage of the new version
Thanks ！

**Comment by altendky** [comment]
There is a readme and a wiki.  Do you have a question about a specific part?

**Reply by leochen12-rgb** [reply]
I'm not sure about the configuration here

```yaml
plotting:
        # Your public keys.  Be sure to use the pool contract address for
        # portable pool plots.  The pool public key is only for original
        # non-portable plots that can not be used with the official pooling
        # protocol.
        # farmer_pk: ...
        # pool_pk: ...
        # pool_contract_address: ...
```

If I want to join a new mine, Do I just need to fill in the pool_contract_address to work？
I'm looking forward to your reply, because I'm using your tool for the first time. 
Thank you very much !

**Reply by altendky** [reply]
At this point you should pretty much always be creating portable plots.  Those are made with the farmer public key and the pool contract address.  The pool contract address comes from the plot NFT that you create.  So, the plots you create are tied to your plot NFT.  You can direct your plot NFT to a pool if you want and you can change which pool it points to later as well.  Maybe take a look through https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide.  plotman is a tool for automating plot creation but doesn't really replace learning about the underlying tech and tools first.

**Reply by DroptheHammer** [reply]
In order to create a portable plot I have to join a pool already?  Can't I just plot portable ones and have them mine just for me while i'm deciding if/when to join a pool? I thought these could be moved or tied to any pool I wanted, but the way the config looks with me entering a pooling address, it makes me fear these are locked to the pool I selected at that point in time.

**Reply by altendky** [reply]
> At this point you should pretty much always be creating portable plots. Those are made with the farmer public key and the pool contract address. The pool contract address comes from the plot NFT that you create. So, the plots you create are tied to your plot NFT. You can direct your plot NFT to a pool if you want and you can change which pool it points to later as well. Maybe take a look through [Chia-Network/chia-blockchain/wiki/Pooling-User-Guide](https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide). plotman is a tool for automating plot creation but doesn't really replace learning about the underlying tech and tools first.

I think I addressed your concerns here.  The "pool contract address" is part of your plot NFT, not a pool.  You can create your plot NFT for solo farming.  You can join, change, and leave pools later as well.
